loaders/src/glTF/babylon.glTFFileLoader.ts

```diff
@@ -67,8 +67,9 @@ module BABYLON {
                 var bufferInput = GLTFUtils.GetBufferFromAccessor(runtime, runtime.gltf.accessors[inputData]);
                 var bufferOutput = GLTFUtils.GetBufferFromAccessor(runtime, runtime.gltf.accessors[outputData]);
 
-                var targetID = channel.target.id;
-                var targetNode: any = runtime.gltf.nodes[channel.target.id].babylonNode;
+                var targetID = channel.target.node;
+                var targetNode: any = runtime.babylonScene.getNodeByID("node" + targetID);

```

Comment:
Consider factoring `"node" + targetID` into a separate function as you use it in multiple places and they need to match